<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Martin Preuss&lt;martin@aquamaniac.de&gt;</author>
 <class>QGuiProgressWidgetUI</class>
 <widget class="QWidget" name="QGuiProgressWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Progress</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="textWidget">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Q3TextBrowser" name="logWidget"/>
   </item>
   <item>
    <widget class="Q3GroupBox" name="currentGroupBox">
     <property name="title">
      <string>Current Action</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="currentLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Current Action</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Q3ProgressBar" name="currentProgressBar"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3ProgressBar" name="progressBar"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel2">
       <property name="text">
        <string>Elapsed time:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>0:00</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer14">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>65</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="textLabel3">
       <property name="text">
        <string>(c) 2007 Martin Preuss</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>65</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="abortButton">
       <property name="text">
        <string>Abort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
  <customwidget>
   <class>Q3ProgressBar</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3ProgressBar</header>
  </customwidget>
  <customwidget>
   <class>Q3TextBrowser</class>
   <extends>Q3TextEdit</extends>
   <header>Qt3Support/Q3TextBrowser</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
